metloom/pointdata/snotel.py

```diff
@@ -44,7 +43,7 @@ def __init__(self, station_id, name, metadata=None):
         self._tzinfo = None
 
     def _snotel_response_to_df(self, result_map: Dict[SensorDescription, List[dict]],
-                               duration: str, include_measurement_date=False):
+                               duration: str, include_measurement_date=False, **kwargs):
```

Comment:
This **kwargs doesn't look like it is getting used

Reply:
thanks, I also needed to remove passing the kwargs to this function, which was being done but not needed

Reply:
@robertson-mark I think this is the last outstanding thing then we should be good to merge

Reply:
thanks, should be good now